resources/js/pack-it-forms.js

```diff
@@ -450,15 +447,18 @@ function escape_pacforms_string(string) {
 
 function bracket_data(data) {
     if (data) {
-        data = data.trim();
-        data = data.replace("`]", "``]");
-        data = data.replace(/([^`])]/, "$1`]");
+        data = data.replace(/`]/g, "``]");
+        data = data.replace(/([^`])]/g, "$1`]");
         return "[" + data + "]";
     } else {
         return null;
     }
 }
 
+function unbracket_data(data) {
+    return data.substring(1, data.length - 1).replace(/`]/g, "]");
+}
```

Comment:
I think this function only gets called from `parse_form_data_text` and I can't think of other places we're likely to need it from. Since it is only one line and the capture of the previous lines is implemented in `parse_form_data_text`, how about just inlining this there too?

Reply:
An automated test should call unbracket_data, to verify it reverses bracket_data.

Reply:
Good point.